packages/typespec-test/package.json

```diff
@@ -3,17 +3,17 @@
   "version": "1.0.0",
   "type": "module",
   "dependencies": {
-    "@typespec/compiler": ">=0.48.0 <1.0.0",
     "@azure-tools/typespec-ts": "workspace:^0.17.1",
-    "@typespec/http": ">=0.48.0 <1.0.0",
-    "@typespec/rest": ">=0.48.0 <1.0.0",
-    "@typespec/openapi": ">=0.48.0 <1.0.0",
-    "@typespec/versioning": ">=0.48.0 <1.0.0",
-    "@azure-tools/typespec-azure-core": ">=0.34.0 <1.0.0",
-    "@azure-tools/typespec-autorest": ">=0.34.0 <1.0.0",
-    "@azure-tools/typespec-client-generator-core": "0.35.0-dev.9",
+    "@typespec/openapi": ">=0.49.0 <1.0.0",
+    "@azure-tools/typespec-autorest": ">=0.35.0 <1.0.0",
     "@typespec/openapi3": ">=0.48.0 <1.0.0",
```

Comment:
update openapi3 to v0.49 now and remvoed the prettier-plugin-typespec, before I used this to format our tsp codes and I think we could remove this because we could do that by `tsp format` command.